db/re/mob_db.txt

```diff
@@ -2630,19 +2630,19 @@
 //3470,WATERMELON_48
 //3471,WATERMELON_49
 //3472,WATERMELON_50
-//3473,AS_RAGGED_GOLEM
-//3474,AS_BLOODY_KNIGHT
-//3475,AS_WIND_GHOST
-//3476,AS_ZOMBIE
-//3477,AS_IMMORTAL_CORPS
-//3478,AS_ZOMBIE_SLAUGHTER
-//3479,AS_ZOMBIE_MASTER
-//3480,AS_CURSED_SOLDIER
-//3481,AS_EVIL_SHADOW1
-//3482,AS_EVIL_SHADOW2
-//3483,AS_EVIL_SHADOW3
-//3484,AS_D_RAGGED_GOLEM
-//3485,AS_D_CURSED_SOLDIER
+3473,AS_RAGGED_GOLEM,Stefan.J.E.Wolf,Stefan.J.E.Wolf,160,20000000,1,1000000,1000000,1,1307,103,146,30,189,25,125,45,104,33,10,12,2,1,89,0x6283695,300,1008,1200,540,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,31026,1
```

Comment:
seems like the monsters didnt have the proper item drop yet.

Reply:
I've created a PR to amend this based on information from Divine-Pride and RMS DBs